test/resources/logback-test.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes I believe so - the `test/resources/logback.xml` squashes application logging when the tests are running, while the `test/resources/logback-test.xml` quietens down the test framework logging.